### PR TITLE
DB-11890 Short-circuit IndexConglomerate.readExternal once the engine has started (3.1)

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Review by carolp-503 — APPROVED** [review]
Approved for 3.1

File: splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/hbase/HBaseConglomerate.java [hunk 1]
```diff
@@ -364,39 +364,6 @@ protected void writeExternalOld(ObjectOutput out) throws IOException {
...
-     * Restore the in-memory representation from the stream.
-     * <p/>
-     *
-     * @throws ClassNotFoundException Thrown if the stored representation
-     *                                is serialized and a class named in
-     *                                the stream could not be found.
-     * @see java.io.Externalizable#readExternal
-     **/
-    @Override
-    public void readExternal(ObjectInput in) throws IOException {
```
Comment (martinrupp) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/hbase/HBaseConglomerate.java:377]: DB-11857 modifies HBaseConglomerate.readExternal, and then DB-11890 delete exactly that function, and i'm not sure how to resolve that. Are you sure HBaseConglomerate.readExternal should be deleted?

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3139/

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/3152/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by cloudspliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**